lib/SILGen/SILGenExpr.cpp

```diff
@@ -2709,7 +2709,8 @@ static SILFunction *getOrCreateKeyPathGetter(SILGenModule &SGM,
   auto signature = SILFunctionType::get(genericSig,
     SILFunctionType::ExtInfo(SILFunctionType::Representation::Thin,
                              /*pseudogeneric*/ false,
-                             /*noescape*/ false),
+                             /*noescape*/ false,
+                             /*clangFunctionType*/ nullptr),
```

Comment:
This specific pattern (thin with no unusual properties) seems to be common enough in SILGen that there probably ought to be a `getThin()` convenience factory for it on `ExtInfo`.  Whether that makes sense e.g. for the uses in ConstraintSystem is a separate question.

Reply:
Oops, forgot to address this. 🤦‍♂  On it.